apollo-router/src/services/supergraph/tests.rs

```diff
@@ -1154,13 +1154,31 @@ async fn root_typename_with_defer_and_empty_first_response() {
         .await
         .unwrap();
 
+    let query = r#"
+        query {
+            ...OnlyTypename
+            ... @defer {
+                currentUser {
+                    activeOrganization {
+                        id
+                        suborga {
+                            id
+                            name
+                        }
+                    }
+                }
+            }
+        }
+
+        fragment OnlyTypename on Query {
+          __typename
+        }
```

Comment:
I wrapped __typename into a fragment here to test my change together with differ

Reply:
Done ✅